bundles/org.openhab.binding.paradoxalarm/src/test/java/main/Main.java

```diff
@@ -68,8 +75,7 @@ public static void main(String[] args) {
     private static void infiniteLoop(ParadoxPanel paradoxSystem, IParadoxCommunicator communicator) {
         try {
             updateDataCache(communicator, false);
-            Thread.sleep(5000);
-            paradoxSystem.updateEntitiesStates();
+            scheduler.schedule(paradoxSystem::updateEntitiesStates, 5000, TimeUnit.MILLISECONDS);

```

Comment:
5 seconds?

Reply:
That's basically the each 5 seconds call to the system to retrieve new states. 5 seconds is OK, thread sleep is also OK here as this is the main class for testing and no other threads will do anything concurent but moved to scheduler just to test if it works also here

Reply:
Sorry that i was a little bit vague in this comment I meant that it make more sense to code it as 5 seconds instead of 5000 msec